### ISSUE TITLE
Non-alpine image

### ISSUE THREAD

**Issue by esetnik** [ngineered/nginx-php-fpm#160]
I need a variant that uses the `FROM php:7.1.7-fpm` base image because I need to install `sqlsrv` pecl package and alpine does not distribute the required `msodbcsql` for alpine.

**Comment by esetnik** [comment]
I took a stab at it, see https://github.com/ngineered/nginx-php-fpm/pull/161

**Comment by richarvey** [comment]
Would you be prepared to maintain this branch for us? If so we can automate the build.

**Comment by esetnik** [comment]
I don’t think it should be a separate branch. If I plan to maintain it, I’d expect it to be part of the main codebase and future PRs would impact both alpine and Debian variants.
